data/categories/vp_vehicles_parts.yml

```diff
@@ -4453,6 +4453,160 @@
   - upholstery_pattern
 - id: vp-2-1
   name: Aircraft
+  children:
+  - vp-2-1-1
+  - vp-2-1-2
+  - vp-2-1-3
+  - vp-2-1-4
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-1
+  name: Commercial Aircraft
+  children:
+  - vp-2-1-1-1
+  - vp-2-1-1-2
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-1-1
+  name: Passenger Jets
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-1-2
+  name: Cargo Aircraft
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-2
+  name: Private Jets
+  children:
+  - vp-2-1-2-1
+  - vp-2-1-2-2
+  - vp-2-1-2-3
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-2-1
+  name: Light Jets
```

Comment:
It seems the terms "private jet" and "business jet" can be used interchangeably (https://en.wikipedia.org/wiki/Business_jet). I suggest we call the parent `Business Jets` instead of "private jets" to avoid focusing on the actual ownership, which can vary without defining the product, and keep two child nodes instead of three: `Light Jets` and `Turboprop Aircraft`.